gk/main.c

```diff
@@ -1386,6 +1618,22 @@ process_cmds_from_mailbox(
         }
 }
 
+static void
+send_pkts(uint8_t port, uint16_t tx_queue,
+	uint16_t num_pkts, struct rte_mbuf **bufs)
```

Comment:
Could you check that this function work when `num_pkts == 0`? Also, add a comment to the function saying that `num_pkts` may be zero.